common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/DownloadFile.java

```diff
@@ -26,22 +26,14 @@
  * which ensures data is not read until after the writer is closed.
  */
 public interface DownloadFile {
-  /**
-   * Delete the file.
-   *
-   * @return  <code>true</code> if and only if the file or directory is
```

Comment:
I think that's probably OK - will likely trigger a MiMa error that will need to be excluded. Nobody should be implementing this interface outside Spark.

Reply:
You're correct - it did cause a MiMa failure.
I've removed the commit. It's probably not worth it.